src/components/PreSurveyForm/PreSurveyForm.tsx

```diff
@@ -81,10 +81,13 @@ const PreSurveyForm = () => {
       },
     });
 
+    const upi = auth.currentUser?.email?.split('@')?.[0];
+
     setUserProperties(analytics, {
       user_gender: values.gender,
       ethnicity: values.ethnicity,
       prev_experience: programmingExperienceLabel,
```

Comment:
Could you move the prev experience question to the end survey, and log it there too

Reply:
Removed PreSurveyForm file, I don't think we need a previous experience question per discussion in #86